workflow-templates/crowdin-download.yml

```diff
@@ -12,18 +12,18 @@ on:
 jobs:
   download-from-crowdin:
     runs-on: [self-hosted, production, medium]
-    container: docker.tw.ee/actions_java15
+    container: docker.tw.ee/actions_java17
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Get branch name
         id: vars
         run: echo ::set-output name=branch_name::${GITHUB_REF##*/}
 
       - name: crowdin action
-        uses: crowdin/github-action@1.3.1
+        uses: crowdin/github-action@v1
```

Comment:
so v1 is more recent than 1.3.1? 🤔

Reply:
Yes. The latest version is 1.13.0 but the [project page](https://github.com/crowdin/github-action) recommends just specifying v1. Presumably this then pulls the latest minor version.

Reply:
More versioning info here: https://github.com/crowdin/github-action